.github/workflows/lib-injection.yml

```diff
@@ -68,5 +75,5 @@ jobs:
         with:
           push: true
           tags: ${{ steps.set_names.outputs.image_name }}
-          platforms: "linux/amd64,linux/arm64/v8"
-          context: ./tests/lib-injection
+          platforms: "linux/amd64"
```

Comment:
We built arm64 for the original sample app since the agent team needed an arm64 image to end-to-end test the library injection stuff on their end. They have that image still available which they use. For these images going forward only we will use them so we can save resources and just built amd64.